dear_petition/petition/export/forms.py

```diff
@@ -108,14 +105,11 @@ def map_header(self):
             self.data["Superior"] = Checkbox("")
 
     def map_file_no(self):
-        if self.petition.offense_records.count() > 1:
-            self.data["ConsJdgmntFileNum"] = self.MULTIPLE_FILE_NO_MSG
-        else:
-            offense_record = self.get_most_recent_record()
-            if offense_record:
-                self.data[
-                    "ConsJdgmntFileNum"
-                ] = offense_record.offense.ciprs_record.file_no
+        offense_record = self.get_most_recent_record()
```

Comment:
Heard back from Ed, his response: " it doesn’t really matter which file is used there, except that it would typically be the first file listed in the body of petition, which could be any file (oldest, most recent, whatever). We do not usually bother to list the files in chronological order, and no one has ever complained."

Updated the PR to grab the first one from the ordered list (since that's the list that is used to fill out all the `Field No` fields)

Reply:
Sounds good — thanks for following up with that!